msm/util.py

```diff
@@ -32,3 +34,13 @@ def wrapper(*args, **kwargs):
             env.update(self.env)
             return super(Git, self).__getattr__(item)(*args, env=env, **kwargs)
         return wrapper
+
+
+class MsmProcessLock(InterProcessLock):
```

Comment:
Why is this lock needed? (In what case does the skills.json get corrupted). If the power got pulled while msm was doing anything would the lock cause msm to fail on subsequent boots?

Reply:
My reasoning is that some of the skill operations are rather slow and now they can occur at any time. If 2 simultaneous installations occurs one could easily be overwritten.

The lock file exists in /tmp/ which should in most cases be reset completely. Also as far as I've been able to read the lock status of fcntl locks are stored in memory and will be cleared on restart.

But I'll do a test later tonight to verify.